src/components/lime-ai-avatar/examples/lime-ai-avatar-basic.tsx

```
@@ -0,0 +1,48 @@
+@Component({
+    tag: 'limel-example-lime-ai-avatar-basic',
+    shadow: true,
+    styleUrl: 'lime-ai-avatar-basic.scss',
+})
+export class LimeAiAvatarBasicExample {
+    @State()
+    private isThinking: boolean = false;
+
+    public render() {
+        return [
+            <div class="avatar-container">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)